kv/main.go

```diff
@@ -585,6 +587,7 @@ func main() {
 				}
 			}
 
+			cumLatency.Merge(h)
```

Comment:
Merge returns a value (number of values "dropped"). Do you know what that is and if we should care?